README.md

```diff
@@ -1 +1,46 @@
-# Leaderboard
\ No newline at end of file
+# Leaderboard
+
+> LeaderBoard
+
+## Built With
+
+- HTML
+- JavaScript
+- CSS
+- webpack
+
+## Live Demo
+
```

Comment:
- [ ] The live demo link is broken. Please either update this with a live deployment link of your project or simply let users know that there is no live demo yet by having a "coming soon" notice in its place 👍